backend/x11/output.c

```diff
@@ -114,9 +114,8 @@ struct wlr_output *wlr_x11_output_create(struct wlr_backend *backend) {
 		wl_list_length(&x11->outputs) + 1);
 	parse_xcb_setup(wlr_output, x11->xcb_conn);
 
-	uint32_t mask = XCB_CW_BACK_PIXEL | XCB_CW_EVENT_MASK;
+	uint32_t mask = XCB_CW_EVENT_MASK;
 	uint32_t values[2] = {
```

Comment:
Can this be changed to a single uint32_t now?

Reply:
Err, we shouldn't specify the size of these arrays explicitly.

Reply:
Good catch. I changed into a variable length array instead.